src/defaults/fileIcons.ts

```diff
@@ -1210,6 +1210,9 @@ const fileIcons: Record<string, {
       '.whitesource',
     ],
   },
+  'json-schema': {
+    fileNames: ['schema.json'],
```

Comment:
```suggestion
    fileNames: ['schema.json'],
    fileExtensions: ['schema.json'],
```
Might be worth adding this to cover `xyz.schema.json`?